src/flightRecorder.cpp

```diff
@@ -1006,6 +1014,7 @@ class Recording {
         writePackages(buf, &lookup);
         writeSymbols(buf, &lookup);
         writeLogLevels(buf);
+        writeUserEventTypes(buf);
```

Comment:
Write them before log levels. There is no difference for parsing, but it will be easier to spot visually if JFR file is complete or truncated, since valid JFR will always end with predefined recognizable list of log levels.